testcharms/charms/dummy-storage-fs/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+name: dummy-storage-fs
+maintainer:
+  - Christopher Lee <chris.lee@canonical.com>
+  - Burton Swan <burton.swan@canonical.com>
+  - Evanson Mwangi <evanson.wainaina@canonical.com>
+summary: Dummy charm that utilises storage.
+description: This dummy-storage-fs charm is used for testing charm storage.
+categories:
+  - misc
+series:
+  - trusty
+  - xenial
+  - artful
+  - bionic
+  - eoan
+  - focal
+  - jammy
```

Comment:
Juju is soon to drop series other than Jammy and Focal, so the others will be redundant

Reply:
This will run in 2.9 at the moment the other series might as well be supported.